packages/fxa-settings/src/components/Settings/PageDisplayName/index.test.tsx

```diff
@@ -103,24 +103,3 @@ it('displays a general error in the alert bar', async () => {
   await submitDisplayName('John Nope');
   expect(settingsContext.alertBarInfo?.error).toBeCalledTimes(1);
 });
-
-it('displays the GraphQL error', async () => {
-  const gqlError: any = new Error('test error');
-  gqlError.graphQLErrors = ['test'];
-  const account = {
-    displayName: 'jrgm',
-    setDisplayName: jest.fn().mockRejectedValue(gqlError),
-  } as unknown as Account;
-  renderWithRouter(
-    <AppContext.Provider value={mockAppContext({ account })}>
-      <PageDisplayName />
-    </AppContext.Provider>
-  );
-  // suppress the console output
-  let consoleErrorSpy = jest
-    .spyOn(console, 'error')
-    .mockImplementationOnce(() => {});
-  await submitDisplayName('Jane Nope');
-  expect(screen.getByTestId('tooltip')).toBeInTheDocument();
-  consoleErrorSpy.mockRestore();
-});
```

Comment:
Deleted because we don't do this anymore?

Reply:
Oh never mind I see the changes below.

Reply:
Yes, removed because we were directly showing any gql error that occurred. For example, I saw an ECONREFUSED error in the tooltip while testing other things. Ideally we might want to filter errors and show some in the tooltip, but the generic error should be sufficient for now. I might file an issue to more selectively show an error in the tooltip if the submitted name does not respect validation criteria.